app/vmalert/config/config_test.go

```diff
@@ -23,6 +24,45 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+func TestParseConfigWithMultiDocNegative(t *testing.T) {
```

Comment:
I think `TestParseConfigWithMultiDocNegative` and `TestParseConfigWithMultiDocPositive` are not necessary here since they are almost identical to `TestParse_Success` and `TestParse_Failure`. WDYT just include your testdata yaml to the later two functions?

Reply:
Sure, let me try and do it